start-tests.sh

```diff
@@ -147,48 +158,64 @@ fi
 
 log_color "yellow" "Setting env to run in:" "$NODE_ENV\n"
 
-log_color "cyan" "Create RBAC users"
-if [ -f /rbac-setup.sh ]; then
-  chmod +x /rbac-setup.sh
-  source /rbac-setup.sh
-else # DEV
-  chmod +x build/rbac-setup.sh
-  source build/rbac-setup.sh
+if [[ -z $CYPRESS_TAGS_INCLUDE ]]; then
+  log_color "purple" "CYPRESS_TAGS_INCLUDE" "not exported; set ${PURPLE}CYPRESS_TAGS_INCLUDE${NC} to include a test tags i.e ${YELLOW}@canary${NC}, if you wish to execute on a subset of tests)"
+else
+  log_color "purple" "Including tests that only contain the following tags: ${YELLOW}$CYPRESS_TAGS_INCLUDE${NC}"
+  CYPRESS_TAGS=$CYPRESS_TAGS_INCLUDE
 fi
 
-echo -e
-
 if [[ -z $CYPRESS_TAGS_EXCLUDE ]]; then
-  log_color "purple" "CYPRESS_TAGS_EXCLUDE" "not exported; running all test (set ${PURPLE}CYPRESS_TAGS_EXCLUDE${NC} to include a test tags i.e ${YELLOW}@critical${NC}, if you wish to exclude a test from the execution)\n"
+  log_color "purple" "CYPRESS_TAGS_EXCLUDE" "not exported; set ${PURPLE}CYPRESS_TAGS_EXCLUDE${NC} to include a test tags i.e ${YELLOW}@critical${NC}, if you wish to exclude a test from the execution)\n"
 else
   log_color "purple" "Excluding tests that contain the following tags: ${YELLOW}$CYPRESS_TAGS_EXCLUDE${NC}"
 fi
 
+if [[ ! -z $CYPRESS_TAGS_INCLUDE || ! -z $CYPRESS_TAGS_EXCLUDE ]]; then
+  CYPRESS_TAGS="$CYPRESS_TAGS_INCLUDE"
+  CYPRESS_TAGS="$CYPRESS_TAGS $CYPRESS_TAGS_EXCLUDE"
```

Comment:
Is this the same as
```
CYPRESS_TAGS="$CYPRESS_TAGS_INCLUDE $CYPRESS_TAGS_EXCLUDE"
```

Reply:
That is correct... adding those changes to the next commit.